migrations/2023-07-26-204110_add_post_view_mode_setting/up.sql

```diff
@@ -0,0 +1,3 @@
+create type post_view_mode_enum as enum ('List', 'Card', 'SmallCard');
+
+alter table local_user add column post_view_mode post_view_mode_enum default 'List' not null;
```

Comment:
Not sure whether this should be called `default_post_view_mode`, or just `post_view_mode`